src/Storefront/Resources/app/storefront/src/scss/component/_offcanvas.scss

```diff
@@ -50,13 +50,8 @@ The sliding direction can be left or right.
         padding: $spacer-sm $spacer;
         text-align: left;
 
-        &:focus {
-            box-shadow: none;
-        }
-
-        &,
-        .icon {
-            color: $gray-600;
+        &:focus-visible {
+            box-shadow: inset 0 0 0 0.125rem $focus-ring-color, inset 0 0 0 0.25rem #fff;
```

Comment:
Can we also add a custom variable for those cases similar to `$focus-ring-box-shadow` which gives you the default focus ring that is not "inset"

Reply:
Also we should not use `#fff` but something like `$body-bg`

Reply:
Done! 👍